discovery/discovery.go

```diff
@@ -106,6 +107,14 @@ func ProvidersFromConfig(cfg config.ServiceDiscoveryConfig) map[string]TargetPro
 	for i, c := range cfg.AzureSDConfigs {
 		app("azure", i, azure.NewDiscovery(c))
 	}
+	for i, c := range cfg.TritonSDConfigs {
+		t, err := triton.New(log.Base(), c)
```

Comment:
I know it's equally wrong in the other SDs, but let's make it `log.With("sd", "triton")` here so all log messages have proper context.